contracts/Swap/lib/TransferHelper.sol

```diff
@@ -26,7 +26,7 @@ library TransferHelper {
         // bytes4(keccak256(bytes('transferFrom(address,address,uint256)')));
         (bool success, bytes memory data) = token.call(abi.encodeWithSelector(0x23b872dd, from, to, value));
         if (!(success && (data.length == 0 || abi.decode(data, (bool))))) {
-            revert TransferFromFailed();
+            revert SafeTransferFromFailed();
         }
     }
 
```

Comment:
Should we rename `safeTransferETH` to `safeTransferBNB`, in the context of N-04?

Reply:
Yes, Fixed, 9751c85d7054485d5f082aa4ba48c7ce878d9a34